core/const.py

```diff
@@ -797,3 +797,62 @@ class ITLRegion(StrEnum):
     "Pre-Payment)",
     "Towns Fund RDEL Payment which is being utilised on TF project related activity",
 ]
+
+TABLE_AND_COLUMN_TO_ORIGINAL_COLUMN_LETTER = {
```

Comment:
Can you add a comment saying this is Towns Fund, and  Round 4 onwards only (I assume that's what it is anyway as ProjectProgress has the new columns?)?

On that note, is this only implemented on R4? Just wondering what'd happen if we ever need to re-ingest Round 3again.

Reply:
would a more descriptive name for this be something like TOWNS_FUND_REPORTING_FORM_COLUMN_MAPPING ?

Reply:
If we tried to reingest round 3 the column letters in Project progress would be wrong. We should think if we want to cover user centric messaging for round 3 or not maybe

Reply:
We don't need to cover use centric messaging for round 3 because users will never upload for it, and if we have to re-upload the files (because of some data quality issue or something) then we'll do it ourselves

Reply:
In that case maybe we should turn of user centered messaging for rounds prior to the current, because it's no use maintaining messaging for reporting rounds that are never going to be used again

Reply:
We've already updated R1, R2, and R3 multiple times, and there's no reason they should be uploaded again; in the case that they are, and that there are errors, then we will need to revert to the original validation messages as they are more useful for developer debugging.

Reply:
remember we will have to re-import R1 - 3 at least once, since we have all that erroneous data still in from the conditional formatting.